kin-devplatform-sdk/src/main/java/kin/devplatform/ConfigurationLocal.java

```diff
@@ -68,6 +70,7 @@ public void setEnvironment(@NonNull KinEnvironment kinEnvironment) {
 		editor.putString(NEW_BLOCKCHAIN_NETWORK_URL_KEY, kinEnvironment.getNewBlockchainNetworkUrl());
 		editor.putString(NEW_BLOCKCHAIN_PASSPHRASE_KEY, kinEnvironment.getNewBlockchainPassphrase());
 		editor.putString(OLD_BLOCKCHAIN_ISSUER_KEY, kinEnvironment.getOldBlockchainIssuer());
+		editor.putString(MIGRATION_SERVICE_URL_KEY, kinEnvironment.getOldBlockchainIssuer());
```

Comment:
wrong value :scream: `kinEnvironment.getOldBlockchainIssuer()`

Reply:
ha ha ha :
that the problem when you are doing copy paste